### ISSUE TITLE
More regular build?

### ISSUE THREAD

**Issue by pierreozoux** [gliderlabs/docker-alpine#270]
Related to:
https://github.com/gliderlabs/docker-alpine/issues/230
https://github.com/gliderlabs/docker-alpine/issues/238

And:
https://github.com/tootsuite/mastodon/pull/2135#issuecomment-295485222

What is your recommendation?
Would it be better to build this image, say once a day/week?
Or should we use `upgrade` inside our images?

Indeed the last push was 2 months ago.
And there are some vulnerabilities, that are now maybe fixed?
https://hub.docker.com/r/library/alpine/tags/3.2/

**Comment by mattaitchison** [comment]
For now upgrade may be the best option.

The vulnerabilities from 3.2 appear to be resolved in [3.5](https://hub.docker.com/r/library/alpine/tags/3.5/).

**Comment by andyshinn** [comment]
I don't think it makes sense to build so often. This would cause all downstream images to pull new bases and do full rebuilds every time there is a new alpine image. But we can try to stay more on top of patch level release (such as 3.5.1, 3.5.2, etc).

I'll try to write some instructions so that anyone can propose a release. A pull request just needs to be opened against the `release` branch and once merged it will cut new builds.
